.github/workflows/build_sqlite_binaries.yml

```
@@ -0,0 +1,107 @@
+name: Build SQLite Binaries
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+  workflow_dispatch:
+
+jobs:
+  build:
```

```suggestion
  sqlite-build:
```

Let's have job name the same name as file name, but as soon as we can't have dots we just remove `.yml`